src/compiler/types.ts

```diff
@@ -1693,6 +1693,8 @@ namespace ts {
           * @param path The path to test.
           */
         fileExists(path: string): boolean;
+
+        readFile(path: string): string;

```

Comment:
> What is the timing of this change? Will it be part of 2.0? If so this will be a challenge for us.

no, this is a TS 2.1 feature. but once checked in it will start showing up in `typescript@next`. This is why i wanted to make sure that all partners are aware of it. I would recommend adding the readFile method now. in TS 2.0 it will not be used; this way users can mix and match tool versions without breakage.